js/screen/app/ctrls/main_ctrl.js

```diff
@@ -6,6 +6,32 @@ TestApp.controllers.controller('MainCtrl', ['$scope', '$location', '$timeout', f
   $scope.total_questions = 0;
   $scope.marked_questions = 0;
   $scope.questions_default = {
+    multi_screen: [
+      {
+        text: 'I have tested that with 2 or more screens connected in a game session,<br />there is always only 1 primary screen.',
```

Comment:
The "I have tested that with" seems redundant and repetitive

Reply:
Its for consistency at this point.
All new test cases we added since last Nov start with **I have tested ...** to ensure that the developers understand that these are their responsibility to test these things before sending in builds.

But you are right, in a more optimal world it would probably be done differently and request that configuration only once (perhaps as a signing procedure when uploading a build) and be able to also show the exact steps to test each of the cases.